nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -58,91 +58,19 @@ define([
             we make the bulk import work with multiple data types
         */
         function buildParamsWidget(node) {
-            const paramBus = runtime
-                .bus()
-                .makeChannelBus({ description: 'Parent comm bus for input widget' });
             // This is the key in the model that maps to the list of params for the current app.
-            const paramKey = `params.${fileType}`;
+            const paramKey = `${fileType}`;
+            const paramBus = buildMessageBus(paramKey, 'Parent comm bus for parameters widget');
+            paramBus.on('parameter-changed', (message) => {
+                updateModelParameterValue(paramKey, PARAM_TYPE, message);
+            });
 
             const widget = ParamsWidget.make({
                 bus: paramBus,
                 workspaceId: runtime.workspaceId(),
-                initialParams: model.getItem(paramKey),
-            });
-
-            paramBus.on('sync-params', (message) => {
```

Comment:
This is all basically duplicated in `buildFilePathWidget`, so those got factored into their own function.